MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java

```diff
@@ -337,7 +337,7 @@ private void drawShadow(Canvas canvas) {
      */
     private void update(boolean force, boolean async) {
         View view = mViewRef.get();
-        if (view != null) {
```

Comment:
I think I'm missing something on this change 😄 If we skip this code block when elevation is 0 or less (less being a bug, probably), then `CompatElevationUpdateRunnable` never runs and `onShadowUpdate` is not called, which means we don't reset the shadow lengths / paints / bitmaps.

So if the elevation is something larger than 0, and then it's set to 0, the reset is not performed correctly. Am I missing something?